pkgs/development/libraries/fastcdr/0002-Load-standard-install-dir-variables.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+Date: Wed, 15 Dec 2021 04:28:08 +0100
+Subject: [PATCH] Load standard install dir variables
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ce8edad..28a6a51 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -22,6 +22,8 @@ if(PROJECT_SOURCE_DIR)
+     set(IS_TOP_LEVEL FALSE)
+ endif()
+ 
++include(GNUInstallDirs)
```

Comment:
This should be upstreamed.

Reply:
I removed the patch as the variables it introduces are not used anyway. It needs more changes, ideally from upstream.